src/bench.h

```diff
@@ -12,6 +12,10 @@
 #include <math.h>
 #include "sys/time.h"
 
+static void escape(void *p) {
```

Comment:
nit: Perhaps rename to memory fence/barrier to make this more clear and add a comment what this is for?

Reply:
Yeah i'll rename and add a comment